stable/nginx-ingress/README.md

```diff
@@ -59,6 +59,7 @@ Parameter | Description | Default
 `controller.scope.namespace` | namespace to watch for ingress | `""` (use the release namespace)
 `controller.extraArgs` | Additional controller container arguments | `{}`
 `controller.kind` | install as Deployment or DaemonSet | `Deployment`
+`controller.daemonset.useHostPort` | If `controller.kind` is `DaemonSet`, this will enable enable `hostPort` for TCP/80 and TCP/443 | false
```

Comment:
double enable: `enable enable`

Reply:
Oups :) fixed that and rebase onto master to solve merge conflict.